sky/packages/sky_services/BUILD.gn

```diff
@@ -2,33 +2,54 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("//mojo/public/dart/rules.gni")
+if (is_fuchsia) {
+  import("//build/dart/dart_package.gni")
+} else {
+  import("//mojo/public/dart/rules.gni")
+}
 
-copy("copy_sky_services_license") {
-  sources = [
-    "//AUTHORS",
-    "//LICENSE",
-  ]
+if (is_fuchsia) {
+  dart_package("sky_services") {
+    infer_package_name = true
+    deps = [
+      "//mojo/dart/packages/mojo",
```

Comment:
Is this part of sky_services?  I would have expected this to be in `package:mojo`.